src/index.html

```diff
@@ -15,8 +29,92 @@
       rel="stylesheet"
       href="./style.css"
     />
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <a
+          href="#"
+          class="header__link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO logo"
+            class="header__img"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#Apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#Samsung"
+                class="nav__link is-active"
```

Comment:
This class should be only for first link, remove it everywhere 
```suggestion
                class="nav__link"
```

Reply:
I thought this function on the first element in figma show us how should work all buttons after hovering over them